docs/steampipe-cloud.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Steampipe Cloud
+sidebar_label: Steampipe Cloud
+---
+
+# Steampipe Cloud is now Turbot Pipes!
+
+![](https://turbot.com/images/blog/2023-07-introducing-turbot-guardrails-and-pipes/pipes_wordmarks.svg)
```

Comment:
What's this?

Reply:
Its the image I stole from the introducing-turbot-guardrails-and-pipes blog post:  ![](https://turbot.com/images/blog/2023-07-introducing-turbot-guardrails-and-pipes/pipes_wordmarks.svg)